server/repository/src/db_diesel/key_value_store.rs

```diff
@@ -34,6 +34,7 @@ pub enum KeyValueType {
     SettingsSyncIntervalSec,
     SettingsSyncCentralServerSiteId,
     SettingsSyncSiteId,
+    SettingsSyncSiteUuid,
```

Comment:
Weirdly, I haven't had to add it to the SQL enum, so I suspect I've done something wrong. Integration test passes though!

Reply:
as per our convo, postgres tests would fail unless it's added to enum

Reply:
https://github.com/openmsupply/open-msupply/pull/388/commits/9fed370ec880aa41d27c6b7b0aed59550149a1ae